diff --git a/src/main/config/sql/current/5.2.1/sequences/KC_SEQ_S2S_USER_ATTACHED_FORM_ID.sql b/src/main/config/sql/current/5.2.1/sequences/KC_SEQ_S2S_USER_ATTACHED_FORM_ID.sql
new file mode 100644
index 0000000..12ef696
--- /dev/null
+++ b/coeus-db/coeus-db-sql/src/main/resources/org/kuali/coeus/coeus-sql/current_mysql/5.2.1/sequences/KC_SEQ_S2S_USER_ATTACHED_FORM_ID.sql
@@ -0,0 +1,2 @@
+CREATE SEQUENCE SEQ_S2S_USER_ATTD_FORM_ID INCREMENT BY 1 START WITH 1 NOCACHE
+/
