diff --git a/src/main/java/org/kuali/kra/award/lookup/AwardLookupableHelperServiceImpl.java b/src/main/java/org/kuali/kra/award/lookup/AwardLookupableHelperServiceImpl.java
index a24b24d..c99d2fd 100644
--- a/src/main/java/org/kuali/kra/award/lookup/AwardLookupableHelperServiceImpl.java
+++ b/src/main/java/org/kuali/kra/award/lookup/AwardLookupableHelperServiceImpl.java
@@ -245,8 +245,14 @@ class AwardLookupableHelperServiceImpl extends KraLookupableHelperServiceImpl {
         AwardPerson principalInvestigator = award.getPrincipalInvestigator();
         if (principalInvestigator != null) {
             if (StringUtils.isNotBlank(principalInvestigator.getPersonId())) {
+                try {
                     final KcPerson inqBo = this.kcPersonService.getKcPersonByPersonId(principalInvestigator.getPersonId());
                     inquiryUrl = super.getInquiryUrl(inqBo, PERSON_ID);
+                }
+                catch (IllegalArgumentException e) {
+                    LOG.info("getPrincipalInvestigatorNameInquiryUrl(Award award): ignoring missing person/entity: "
+                            + principalInvestigator.getPersonId());
+                }
             } else {
                 if (principalInvestigator.getRolodexId() != null) {
                     Rolodex inqBo = new Rolodex();
