diff --git a/src/main/webapp/unitHierarchy.jsp b/src/main/webapp/unitHierarchy.jsp
index 3240e51..2dac5ef 100644
--- a/coeus-webapp/src/main/webapp/unitHierarchy.jsp
+++ b/coeus-webapp/src/main/webapp/unitHierarchy.jsp
@@ -14,7 +14,7 @@
  limitations under the License.
 --%>
 <%@ include file="/WEB-INF/jsp/kraTldHeader.jsp"%>
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
+<!DOCTYPE html>
 <html>
 <head>
     <meta http-equiv="content-type" content="text/html; charset=utf-8">
