diff --git a/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherAwards.tag b/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherAwards.tag
index b6fbde8..3b743da 100644
--- a/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherAwards.tag
+++ b/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherAwards.tag
@@ -18,8 +18,8 @@
 <channel:portalChannelTop channelTitle="Awards" />
 <div class="body">
 	<ul class="chan">
-	  <li>Awards in Progress</li>
       <li><portal:portalLink displayTitle="false" title="All my Awards" url="${ConfigProperties.application.url}/kr/lookup.do?methodToCall=search&businessObjectClassName=org.kuali.kra.award.home.Award&docFormKey=88888888&includeCustomActionUrls=true&returnLocation=${ConfigProperties.application.url}/portal.do&projectPersons.fullName=${UserSession.person.firstName}*${UserSession.person.lastName}&hideReturnLink=true">All my Awards</portal:portalLink></li>
+      <li><portal:portalLink displayTitle="false" title="Award Report Tracking" url="${ConfigProperties.application.url}/reportTrackingLookup.do?methodToCall=start&businessObjectClassName=org.kuali.kra.award.paymentreports.awardreports.reporting.ReportTracking&docFormKey=88888888&returnLocation=${ConfigProperties.application.url}/portal.do">Award Report Tracking</portal:portalLink></li>
 	</ul>
 </div>
 <channel:portalChannelBottom />
