diff --git a/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherQuicklinks.tag b/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherQuicklinks.tag
index c034a2c..c84be0f 100644
--- a/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherQuicklinks.tag
+++ b/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherQuicklinks.tag
@@ -18,10 +18,7 @@
 <channel:portalChannelTop channelTitle="Quicklinks" />
 <div class="body">
   <ul class="chan">  	  
-      <li><portal:portalLink displayTitle="true" title="Kc Pessimistic Lock" url="${ConfigProperties.application.url}/kr/lookup.do?methodToCall=start&businessObjectClassName=org.kuali.rice.krad.document.authorization.PessimisticLock&returnLocation=${ConfigProperties.application.url}/portal.do&hideReturnLink=true&docFormKey=88888888" /></li>
       <li><portal:portalLink displayTitle="true" title="Grants.gov Opportunity Lookup" url="${ConfigProperties.application.url}/kr/lookup.do?methodToCall=start&businessObjectClassName=org.kuali.kra.s2s.bo.S2sOpportunity&returnLocation=${ConfigProperties.application.url}/portal.do&hideReturnLink=true&docFormKey=88888888" /></li>
-      <li>Change Password</li>
-      <li><portal:portalLink displayTitle="false" title='Reporting' url='${ConfigProperties.application.url}/reporting.do?methodToCall=getReportParametersFromDesign'>Reporting</portal:portalLink></li>
   </ul>
 </div>
 <channel:portalChannelBottom />
