diff --git a/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherPersonnel.tag b/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherPersonnel.tag
index 5ef7997..1dad6c0 100644
--- a/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherPersonnel.tag
+++ b/src/main/webapp/WEB-INF/tags/rice-portal/channel/researcherPersonnel.tag
@@ -18,10 +18,7 @@
 <channel:portalChannelTop channelTitle="Personnel" />
 <div class="body">
   <ul class="chan">
-    <li>Degree Information</li>
    <li><portal:portalLink displayTitle="false" title="Current & Pending Support" url="${ConfigProperties.application.url}/currentOrPendingReport.do">Current &amp; Pending Support</portal:portalLink></li>
-    <li>Bio-sketches</li>
-    <li>All my Training</li>
   </ul>
 </div>
 <channel:portalChannelBottom />
