diff --git a/src/main/webapp/WEB-INF/tags/rice-portal/channel/unitQuicklinks.tag b/src/main/webapp/WEB-INF/tags/rice-portal/channel/unitQuicklinks.tag
index a4ab10a..d5121e2 100644
--- a/coeus-webapp/src/main/webapp/WEB-INF/tags/rice-portal/channel/unitQuicklinks.tag
+++ b/coeus-webapp/src/main/webapp/WEB-INF/tags/rice-portal/channel/unitQuicklinks.tag
@@ -23,7 +23,6 @@
     <li><portal:portalLink displayTitle="true" title="Grants.gov Opportunity Lookup" url="${ConfigProperties.application.url}/kr/lookup.do?methodToCall=start&businessObjectClassName=org.kuali.kra.s2s.bo.S2sOpportunity&docFormKey=88888888&returnLocation=${ConfigProperties.application.url}/portal.do&hideReturnLink=true" /></li>
     <li><portal:portalLink displayTitle="true" title="Address Book" url="${ConfigProperties.application.url}/kr/lookup.do?methodToCall=start&businessObjectClassName=org.kuali.kra.bo.Rolodex&docFormKey=88888888&returnLocation=${ConfigProperties.application.url}/portal.do&hideReturnLink=true" /></li>
     <li><portal:portalLink displayTitle="true" title="Sponsor Lookup" url="${ConfigProperties.application.url}/kr/lookup.do?methodToCall=start&businessObjectClassName=org.kuali.kra.bo.Sponsor&docFormKey=88888888&returnLocation=${ConfigProperties.application.url}/portal.do&hideReturnLink=true" /></li>
-    <li>Change Password</li>
     <li><portal:portalLink displayTitle="true" title="Keyword Lookup" url="${ConfigProperties.application.url}/kr/lookup.do?methodToCall=start&businessObjectClassName=org.kuali.kra.bo.ScienceKeyword&docFormKey=88888888&returnLocation=${ConfigProperties.application.url}/portal.do&hideReturnLink=true" /></li>
   
     <li><portal:portalLink displayTitle="false" title="Perform Person Mass Change" url="${ConfigProperties.application.url}/personMassChangeHome.do?methodToCall=docHandler&command=initiate&docTypeName=PersonMassChangeDocument">Perform Person Mass Change</portal:portalLink></li>
