diff --git a/src/main/webapp/WEB-INF/jsp/meeting/IacucMeetingManagement.jsp b/src/main/webapp/WEB-INF/jsp/meeting/IacucMeetingManagement.jsp
index f0ed301..a006696 100644
--- a/coeus-webapp/src/main/webapp/WEB-INF/jsp/meeting/IacucMeetingManagement.jsp
+++ b/coeus-webapp/src/main/webapp/WEB-INF/jsp/meeting/IacucMeetingManagement.jsp
@@ -19,8 +19,8 @@
 <%-- Is the screen an inquiry? --%>
 <c:set var="_isInquiry"
     value="${requestScope[Constants.PARAM_MAINTENANCE_VIEW_MODE] eq Constants.PARAM_MAINTENANCE_VIEW_MODE_INQUIRY}" />
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
-<html:html>
+<!DOCTYPE html>
+<html>
 
     <head>
 <c:if test="${not empty SESSION_TIMEOUT_WARNING_MILLISECONDS}">
@@ -265,4 +265,4 @@ jQuery(function(){
 
     </SCRIPT> 
 
-</html:html>
+</html>
