diff --git a/src/main/java/org/kuali/kra/subawardReporting/printing/xmlstream/SubAwardFDPPrintXmlStream.java b/src/main/java/org/kuali/kra/subawardReporting/printing/xmlstream/SubAwardFDPPrintXmlStream.java
index 41f8e2b..9826fcb 100644
--- a/src/main/java/org/kuali/kra/subawardReporting/printing/xmlstream/SubAwardFDPPrintXmlStream.java
+++ b/src/main/java/org/kuali/kra/subawardReporting/printing/xmlstream/SubAwardFDPPrintXmlStream.java
@@ -394,6 +394,7 @@ public class SubAwardFDPPrintXmlStream implements XmlStream  {
                 rolodexDetails.setEmail(subaward.getRolodex().getEmailAddress());
             }
             subcontractDetail.setPONumber(subaward.getPurchaseOrderNum());
+            subcontractDetail.setSubContractCode(subaward.getSubAwardCode());
             if(subaward.getOrganization() != null){
                 subcontractDetail.setSubcontractorName(subaward.getOrganization().getOrganizationName());
                 rolodexDetailsType.setAddress1(subaward.getOrganization().getRolodex().getAddressLine1());
