diff --git a/src/main/java/org/kuali/kra/common/committee/lookup/CommitteeLookupableHelperServiceImplBase.java b/src/main/java/org/kuali/kra/common/committee/lookup/CommitteeLookupableHelperServiceImplBase.java
index 972897a..93f5d0f 100644
--- a/src/main/java/org/kuali/kra/common/committee/lookup/CommitteeLookupableHelperServiceImplBase.java
+++ b/src/main/java/org/kuali/kra/common/committee/lookup/CommitteeLookupableHelperServiceImplBase.java
@@ -128,6 +128,7 @@ public abstract class CommitteeLookupableHelperServiceImplBase<CMT extends Commi
     
     protected abstract String getHtmlAction();
     protected abstract String getDocumentTypeName();
+    protected abstract String getCustomResumeEditUrl(final String editCommitteeDocId);
     
     protected String getKeyFieldName() {
         return "committeeId";
@@ -151,7 +152,7 @@ public abstract class CommitteeLookupableHelperServiceImplBase<CMT extends Commi
                 AnchorHtmlData htmlData = (AnchorHtmlData) htmlDataList.get(0);
                 CD document = ((CMT) businessObject).getCommitteeDocument();
                 String workflowUrl = getKualiConfigurationService().getPropertyValueAsString(KRADConstants.WORKFLOW_URL_KEY);
-                htmlData.setHref(String.format(DOCHANDLER_LINK, workflowUrl, editCommitteeDocId));
+                htmlData.setHref(getCustomResumeEditUrl(editCommitteeDocId));
                 htmlData.setDisplayText("resume edit");
             }
         }
