diff --git a/src/main/config/sql/current_mysql/5.2.1/sequences/KC_SEQ_S2S_USER_ATTACHED_FORM_ATT_ID.sql b/src/main/config/sql/current_mysql/5.2.1/sequences/KC_SEQ_S2S_USER_ATTACHED_FORM_ATT_ID.sql
new file mode 100644
index 0000000..29049c5
--- /dev/null
+++ b/src/main/config/sql/current_mysql/5.2.1/sequences/KC_SEQ_S2S_USER_ATTACHED_FORM_ATT_ID.sql
@@ -0,0 +1,10 @@
+DELIMITER /
+
+CREATE TABLE SEQ_S2S_USER_ATTD_FORM_ATT_ID (
+		id bigint(19) not null auto_increment, primary key (id)
+) ENGINE MyISAM
+/
+ALTER TABLE SEQ_S2S_USER_ATTD_FORM_ATT_ID auto_increment = 1
+/
+
+DELIMITER ;
