diff --git a/src/main/java/org/kuali/kra/budget/printing/xmlstream/IndustrialBudgetXmlStream.java b/src/main/java/org/kuali/kra/budget/printing/xmlstream/IndustrialBudgetXmlStream.java
index bcc6d65..b055ef5 100644
--- a/src/main/java/org/kuali/kra/budget/printing/xmlstream/IndustrialBudgetXmlStream.java
+++ b/src/main/java/org/kuali/kra/budget/printing/xmlstream/IndustrialBudgetXmlStream.java
@@ -339,6 +339,7 @@ public class IndustrialBudgetXmlStream extends BudgetBaseStream {
 			}
 			ReportType reportType = getReportTypeForNonPersonnel(categoryDesc,
 					costElementDesc, calculatedCost, null);
+	         if(calculatedCost.doubleValue()>0.0d)
 	             reportTypeList.add(reportType);
 		}
 		setReportTypeForIndustrialBudgetNonPersonnel(reportTypeList);
