diff --git a/src/main/webapp/WEB-INF/jsp/coi/FinancialEntityEditNew.jsp b/src/main/webapp/WEB-INF/jsp/coi/FinancialEntityEditNew.jsp
index 13fea6f..36957be 100644
--- a/coeus-webapp/src/main/webapp/WEB-INF/jsp/coi/FinancialEntityEditNew.jsp
+++ b/coeus-webapp/src/main/webapp/WEB-INF/jsp/coi/FinancialEntityEditNew.jsp
@@ -16,8 +16,8 @@
 
 <%@ include file="/WEB-INF/jsp/kraTldHeader.jsp"%>
 
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
-<html:html>
+<!DOCTYPE html>
+<html>
 
     <head>
         <script>var jsContextPath = "${pageContext.request.contextPath}";</script>
@@ -184,4 +184,4 @@
 <script language="javascript" src="dwr/interface/SponsorService.js"></script>
 <script language="javascript" src="dwr/interface/StateService.js"></script>
 
-</html:html>
+</html>
