diff --git a/src/main/webapp/WEB-INF/jsp/coi/FinancialEntityView.jsp b/src/main/webapp/WEB-INF/jsp/coi/FinancialEntityView.jsp
index 06f4ea2..cb95d0b 100644
--- a/coeus-webapp/src/main/webapp/WEB-INF/jsp/coi/FinancialEntityView.jsp
+++ b/coeus-webapp/src/main/webapp/WEB-INF/jsp/coi/FinancialEntityView.jsp
@@ -1,8 +1,8 @@
 <%@ include file="/WEB-INF/jsp/kraTldHeader.jsp"%>
 
 
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
-<html:html>
+<!DOCTYPE html>
+<html>
    <head>
 <link rel="stylesheet" href="css/jquery/new_kuali.css" type="text/css" />
 <link rel="stylesheet" href="css/jquery/kuali-stylesheet.css"
@@ -121,4 +121,4 @@
 		</tr>
 	</c:if>
 </table>
-</html:html>
+</html>
