diff --git a/src/main/resources/org/kuali/kra/printing/stylesheet/FDP_Modification_Template.xsl b/src/main/resources/org/kuali/kra/printing/stylesheet/FDP_Modification_Template.xsl
index 1d8dafa..56d8339 100644
--- a/coeus-impl/src/main/resources/org/kuali/kra/printing/stylesheet/FDP_Modification_Template.xsl
+++ b/coeus-impl/src/main/resources/org/kuali/kra/printing/stylesheet/FDP_Modification_Template.xsl
@@ -347,7 +347,7 @@
 												<fo:block/>
 												<xsl:for-each select="subcontract:SubContractData">
 													<xsl:for-each select="subcontract:SubcontractDetail">
-														<xsl:for-each select="subcontract:PONumber">
+														<xsl:for-each select="subcontract:SubContractCode">
 															<xsl:variable name="value-of-template">
 																<xsl:apply-templates/>
 															</xsl:variable>
