diff --git a/src/main/webapp/WEB-INF/tags/budget/budgetVersions.tag b/src/main/webapp/WEB-INF/tags/budget/budgetVersions.tag
index f1c064f..3913cec 100644
--- a/coeus-webapp/src/main/webapp/WEB-INF/tags/budget/budgetVersions.tag
+++ b/coeus-webapp/src/main/webapp/WEB-INF/tags/budget/budgetVersions.tag
@@ -160,7 +160,7 @@
 			<kra:section permission="addBudget">
 			<tr class="addline">
             	<th width="50" align="right" scope="row"><div align="right">Add:</div></th>
-            	<td class="infoline"><label><html:text name="KualiForm" property="newBudgetVersionName" size="16"/></label></td>
+            	<td class="infoline"><label><html:text name="KualiForm" property="newBudgetVersionName" size="16" maxlength="40"/></label></td>
 	            <td class="infoline">&nbsp;</td>
 	            <td class="infoline">&nbsp;</td>
 	            <td class="infoline">&nbsp;</td>
