diff --git a/src/main/webapp/WEB-INF/jsp/multicampus_dummy_login.jsp b/src/main/webapp/WEB-INF/jsp/multicampus_dummy_login.jsp
index 8d04721..691513d 100644
--- a/src/main/webapp/WEB-INF/jsp/multicampus_dummy_login.jsp
+++ b/src/main/webapp/WEB-INF/jsp/multicampus_dummy_login.jsp
@@ -28,50 +28,6 @@
             <script language="JavaScript" type="text/javascript" src="${ConfigProperties.application.url}/${fn:trim(javascriptFile)}"></script>
         </c:if>
     </c:forEach>
-
-    <style type="text/css">
-        div.body {
-            background-image: url("${ConfigProperties.application.url}/rice-portal/images/os-guy.gif");
-            background-repeat: no-repeat;
-            padding-top: 5em;
-        }
-
-        table#login {
-            margin: auto;
-            background-color: #dfdda9;
-            border: .5em solid #fffdd8;
-            /* simple rounded corners for mozilla & webkit */
-            -moz-border-radius: 10px;
-            -webkit-border-radius: 10px;
-        }
-
-        table#login th {
-            height: 30 px;
-            padding-top: .8em;
-            padding-bottom: .8em;
-            color: #a02919;
-            font-size: 2em;
-        }
-
-        #login td {
-            padding: .2em;
-            height: 20px;
-        }
-
-        #login .rightTd {
-            padding-right: 1.2em;
-        }
-
-        #login .leftTd {
-            padding-left: 1.2em;
-        }
-
-        table#login td#buttonRow {
-            padding-top: 1em;
-            padding-bottom: .6em;
-        }
-
-    </style>
   </head>
 
 <body OnLoad="document.loginForm.__login_user.focus();">
