diff --git a/src/main/java/org/kuali/kra/iacuc/actions/IacucProtocolActionsAction.java b/src/main/java/org/kuali/kra/iacuc/actions/IacucProtocolActionsAction.java
index 1ea120f..3558a0c 100644
--- a/coeus-impl/src/main/java/org/kuali/kra/iacuc/actions/IacucProtocolActionsAction.java
+++ b/coeus-impl/src/main/java/org/kuali/kra/iacuc/actions/IacucProtocolActionsAction.java
@@ -167,6 +167,12 @@ public class IacucProtocolActionsAction extends IacucProtocolAction {
             protocolForm.getActionHelper().setCurrentTask("");
         }
         ActionForward actionForward = super.execute(mapping, form, request, response);
+
+        if ("close".equals(protocolForm.getMethodToCall()) || protocolForm.getMethodToCall() == null) {
+            // If we're closing, we can just leave right here.
+            return mapping.findForward(KRADConstants.MAPPING_PORTAL);
+        }
+
         protocolForm.getActionHelper().prepareView();
         protocolForm.getActionHelper().initFilterDatesView();
         // submit action may change "submission details", so re-initialize it
