diff --git a/src/main/java/org/kuali/kra/irb/actions/ProtocolProtocolActionsAction.java b/src/main/java/org/kuali/kra/irb/actions/ProtocolProtocolActionsAction.java
index 97219e4..95a77ea 100644
--- a/coeus-impl/src/main/java/org/kuali/kra/irb/actions/ProtocolProtocolActionsAction.java
+++ b/coeus-impl/src/main/java/org/kuali/kra/irb/actions/ProtocolProtocolActionsAction.java
@@ -173,6 +173,11 @@ public class ProtocolProtocolActionsAction extends ProtocolAction implements Aud
             protocolForm.getActionHelper().setCurrentTask("");
         }
         ActionForward actionForward = super.execute(mapping, form, request, response); 
+        
+        if ("close".equals(protocolForm.getMethodToCall()) || protocolForm.getMethodToCall() == null) {
+            // If we're closing, we can just leave right here.
+            return mapping.findForward(KRADConstants.MAPPING_PORTAL);
+        }
         protocolForm.getActionHelper().prepareView();
         // submit action may change "submission details", so re-initializa it
         protocolForm.getActionHelper().initSubmissionDetails();
