diff --git a/src/main/java/org/kuali/kra/coi/disclosure/CoiDisclosureServiceImpl.java b/src/main/java/org/kuali/kra/coi/disclosure/CoiDisclosureServiceImpl.java
index 73129cc..460f83b 100644
--- a/coeus-impl/src/main/java/org/kuali/kra/coi/disclosure/CoiDisclosureServiceImpl.java
+++ b/coeus-impl/src/main/java/org/kuali/kra/coi/disclosure/CoiDisclosureServiceImpl.java
@@ -15,6 +15,8 @@
  */
 package org.kuali.kra.coi.disclosure;
 
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
 import org.apache.commons.collections.CollectionUtils;
 import org.apache.commons.lang.StringUtils;
 import org.kuali.kra.award.awardhierarchy.AwardHierarchy;
@@ -78,6 +80,7 @@ import java.util.*;
  * This class...
  */
 public class CoiDisclosureServiceImpl implements CoiDisclosureService {
+    private static final Log LOG = LogFactory.getLog(CoiDisclosureServiceImpl.class);
 
     private BusinessObjectService businessObjectService;
     private KcPersonService kcPersonService;
@@ -233,7 +236,8 @@ public class CoiDisclosureServiceImpl implements CoiDisclosureService {
     private DisclosurePersonUnit createLeadUnit(String personId) {
 
         DisclosurePersonUnit leadUnit = null;
-        KcPerson kcPerson = kcPersonService.getKcPersonByPersonId(personId);
+        try {
+            final KcPerson kcPerson = kcPersonService.getKcPersonByPersonId(personId);
             if (kcPerson != null && kcPerson.getUnit() != null) {
                 leadUnit = new DisclosurePersonUnit();
                 leadUnit.setLeadUnitFlag(true);
@@ -241,6 +245,10 @@ public class CoiDisclosureServiceImpl implements CoiDisclosureService {
                 leadUnit.setUnitName(kcPerson.getUnit().getUnitName());
                 leadUnit.setPersonId(personId);
             }
+        }
+        catch (IllegalArgumentException e) {
+            LOG.info("createLeadUnit: ignoring missing person/entity: " + personId);
+        }
         return leadUnit;
     }
     
